includes/class-wp-job-manager-stats.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * File containing the class WP_Job_Manager_Stats
+ *
+ * @package wp-job-manager
+ */
+
```

Comment:
Don't we want to put this into a namespace?

Not sure if we'll have more classes there, but I think it is worth asking. :)